src/patternfly/sass-utilities/mixins.scss

```diff
@@ -152,3 +152,70 @@
   scrollbar-width: none; // hides scrollbars in Firefox 64 and up
   -ms-overflow-style: -ms-autohiding-scrollbar; // auto hides scrollbars in Edge
 }
+
+@mixin pf-hidden-visible($val: "block") {
```

Comment:
Were you thinking of updating the other components that use `display: none; visibility: hidden` within this PR?

Reply:
oh nah, I wasn't planning on it. this mixin was just for the hidden/visible functionality.